src/plugins/dev_tools/public/types.ts

```diff
@@ -6,10 +6,8 @@
  * Side Public License, v 1.
  */
 
-import { AppNavLinkStatus } from '@kbn/core/public';
-
 export interface ConfigSchema {
   deeplinks: {
-    navLinkStatus: keyof typeof AppNavLinkStatus;
+    navLinkStatus: 'default' | 'visible';
```

Comment:
I left this navLinkStatus config to not break Cloud deployment using it. It could be deprecated in a following work as I don't think it is being used anymore.